code/modules/jobs/job_types/quartermaster.dm

```diff
@@ -8,6 +8,8 @@
 	//supervisors = "the head of personnel" //ORIGINAL
 	supervisors = "the captain" //SKYRAT EDIT CHANGE
 	selection_color = "#d7b088"
+	exp_requirements = 180 //SKYRAT EDIT CHANGE
```

Comment:
How much time is that?

Reply:
40 hours, exactly the same as every other timelocked Head job. I'm not exactly sure why it's labeled as 180, I snipped the values from the other jobs' .dm